pkgs/development/libraries/yaml-cpp/default.nix

```diff
@@ -3,36 +3,19 @@
 , fetchFromGitHub
 , gitUpdater
 , cmake
-, fetchpatch
 }:
 
 stdenv.mkDerivation rec {
   pname = "yaml-cpp";
-  version = "0.7.0";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "jbeder";
     repo = "yaml-cpp";
-    rev = "yaml-cpp-${version}";
-    hash = "sha256-2tFWccifn0c2lU/U1WNg2FHrBohjx8CXMllPJCevaNk=";
+    rev = version;
```

Comment:
Please also drop the `rev-prefix = "yaml-cpp-"` down in `passthru.updateScript`.